luigi_pipeline/lib/model/seqr_mt_schema.py

```diff
@@ -18,6 +18,24 @@ def __init__(self, *args, ref_data, clinvar_data, hgmd_data, **kwargs):
     def vep(self):
         return self.mt.vep
 
+    @row_annotation()
+    def rsid(self):
+        return self.mt.rsid
+
+    @row_annotation()
+    def filters(self):
+        return self.mt.filters
+
+    @row_annotation()
+    def aIndex(self):
```

Comment:
@hanars named aIndex for conformity, but it is unusual since everything else is snake case. Do you use this?

Reply:
This is not used, we can just get rid of it

Reply:
Sweet, we'll make note of that, but will keep it here for now in case we do any strict comparison validations.